components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1668,10 +1681,20 @@
 <se_nsplit dyn_target="preqx" hgrid="ne0np4_twpx4v1" > 4 </se_nsplit>
 <se_nsplit dyn_target="preqx"> 2 </se_nsplit>
 
+<!-- ne45 defaults to new time step interface -->
+<se_tstep hgrid="ne45np4">  200 </se_tstep>
+<dt_remap_factor  dyn_target="preqx" hgrid="ne45np4"> 2 </dt_remap_factor>
+<dt_tracer_factor dyn_target="preqx" hgrid="ne45np4"> 1 </dt_tracer_factor>
+<qsplit    dyn_target="preqx" hgrid="ne45np4"> -1 </qsplit>
+<rsplit    dyn_target="preqx" hgrid="ne45np4"> -1 </rsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne45np4"> -1 </se_nsplit>
```

Comment:
this block of XML breaks our convention of preqx using the old interface and theta-l using the new interface.   just wanted to make sure it was done intentionally.

Reply:
Yea, this was done intentionally to make it clear that the "old" style defaults are being overridden for ne45, but I can move them to be more consistent.